### PR TITLE
Fixes #510: Improve infinite pagination in media wall

### LINKED ISSUE #510: Improve infinite pagination in media wall
Pagination in media wall can be implemented using `since` and `until` field of loklak search.json API.

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**
- Pagination using `source=twitter`
- Auto-refresh in 10 seconds to avoid too many feeds

**Screenshots (if appropriate)** 

**Link to live demo (if appropriate):** 

Scroll to bottom to get more feeds. Also wait for 30 seconds for auto-refresh.

https://achint08.github.io/loklak_search/wall?query=fossasia

**Closes #510**

### CODE REVIEW

**Comment by Achint08** [comment]
Please review @mariobehling @hemantjadon @singhpratyush @vibhcool @kavithaenair @djmgit @SKrPl .

File: src/app/utils/remove-duplicate.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { ApiResponseResult } from '../models/api-response';
+
+export function removeDuplicate(feeds: ApiResponseResult[], testFeeds: ApiResponseResult[]): ApiResponseResult[] {
```
Comment (hemantjadon) [inline src/app/utils/remove-duplicate.ts]: @Achint08 this should not be here, the utils functions, are the functions, which are generic, to all array types, make a generic function, which doesn't depend on **ApiResponseResult** then use it.

**Comment by Achint08** [comment]
@hemantjadon Done!

**Comment by Achint08** [comment]
Sorry @mariobehling . I forgot the edit the details. No `since` and `until` modifier is used now. API requests are sent using `source=twitter`

**Pull request merged**